app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -56,60 +56,46 @@ keywords:
 ---
 
 
-If you’re applying for teaching jobs in England as a non-UK citizen, you will need a visa or immigration status allowing you to work in England.
+If you’re applying for teaching jobs in England as a non-UK citizen, you will need a [visa or immigration status allowing you to work in England](#check-your-status).
 
-The information on this page is for:
+## How to apply for your visa
 
-* non-UK citizens living in the UK
-* non-UK citizens living outside the UK
+Most non-UK citizens who don't already have a visa or immigration status allowing them to work in England will need a Skilled Worker visa to teach in England. 
 
-You will not need a visa if you:
+### To apply for your visa, follow these steps
```

Comment:
Could you add step numbers to help guide the user. I was expecting to see 'Step 1:, Step 2, etc'

Reply:
Yeah that could help. Would need to add to other page too but that's no problem